pontis-ui/src/components/Protocol/ProtocolCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+export const protocolCategories = [
```

Comment:
This should maybe move to `ProtocolSection` to have content all in one place?

Reply:
Agree, could also have file for data specifically

Reply:
Oh that might be a really good idea.